app/controllers/matches_controller.rb

```diff
@@ -1,10 +1,13 @@
 class MatchesController < ApplicationController
-  before_action :active_staff, :same_organization?
+  before_action :set_pet, only: %i[create]
+  before_action :set_match, only: %i[destroy]
 
   before_action :set_pet, only: %i[create]
   before_action :set_match, only: %i[destroy]
 
   def create
+    authorize! context: {organization: @pet.organization}
```

Comment:
just curious - could `authorize_scope` be used here?

Reply:
Yup, we could use `authorized_scope` in any action really.

In some ways, I think scoping everywhere would be ultimate security. That could potentially avoid ever loading unauthorized data, rather than grabbing data and then denying access. But I have read that it is an easy path to lower performance. Because of this, I opted only to use scoping for `index` actions only.

Reply:
oh yeah that makes sense